### PR TITLE
clean up concurrent adds that were done by mistake

### PR DESCRIPTION
**Pull request by teryhill**
clean up concurrent adds that were done by mistake

### CODE REVIEW

File: interface/forms/annotate_diagram/mapdiagram/C_AbstractAnnotate.php [hunk 1]
```diff
@@ -52,11 +48,11 @@
      *  template module name, passed to Controller's initializer.
      */
     function C_AbstractAnnotate($template_mod = "annotate") {
-    	parent::__construct();
-    	$returnurl = $GLOBALS['concurrent_layout'] ? 'encounter_top.php' : 'patient_encounter.php';
-    	$this->template_mod = $template_mod;
-    	$this->template_dir = $GLOBALS['fileroot'] . "/interface/forms/annotate_diagram/mapdiagram/template/";
-    	
+        parent::__construct();
+        $returnurl = 'encounter_top.php' ;
```
Comment (aethelwulffe) [inline interface/forms/annotate_diagram/mapdiagram/C_AbstractAnnotate.php]: This looks like the line in this file, right?

File: interface/forms/prior_auth/C_FormPriorAuth.class.php [hunk 2]
```diff
@@ -12,59 +12,59 @@ class C_FormPriorAuth extends Controller {
     var $style;
     var $prior_auth;
     var $view;                    
-	
+    
     function __construct($template_mod = "general") {
         parent::__construct();
-        $returnurl = $GLOBALS['concurrent_layout'] ? 'encounter_top.php' : 'patient_encounter.php';
```
Comment (aethelwulffe) [inline interface/forms/prior_auth/C_FormPriorAuth.class.php]: And here...

File: interface/forms/vitals/C_FormVitals.class.php [hunk 3]
```diff
@@ -21,7 +21,7 @@ class C_FormVitals extends Controller {
 
     function __construct($template_mod = "general") {
         parent::__construct();
-        $returnurl = $GLOBALS['concurrent_layout'] ? 'encounter_top.php' : 'patient_encounter.php';
+        $returnurl = 'encounter_top.php';
```
Comment (aethelwulffe) [inline interface/forms/vitals/C_FormVitals.class.php]: And here.
Just four lines of real changes, just the $returnurl that is getting fixed, right?

**Comment by teryhill** [comment]
Yes just the 4 lines

**Comment by aethelwulffe** [comment]
Sorry if this is too hot-button for anyone, but this is a seriously simple PR, and it is **_code that has already previously been merged_**...so I am applying this fix.

**Pull request merged**